src/types/events/message-events.ts

```diff
@@ -67,6 +76,104 @@ export interface BotMessageEvent {
   thread_ts?: string;
 }
 
+interface ChannelArchiveMessageEvent {
```

Comment:
The other interfaces in this file are exported. Should we follow suit with the new ones, as well?

Reply:
Thanks! Fixed.